frontend/src/components/common/edit-user/edit-user.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from 'react';
```

Comment:
Rename it to the `edit-user-popup`, wrap it into `Modal` component and pass `isOpen` prop from the parent